### DISCUSSION TITLE
black page start Kali linux M1 Macbook air  issue

### DISCUSSION THREAD

**Discussion by VMGoodL** [utmapp/UTM#4527] [UTM Help (macOS)] [answered]
**Configuration (required)**
* UTM Version: ARM64
* OS Version: Moterey 12.5
* Device Model: Macbook air 2020 M1 
* Is it jailbroken (name jailbreak used)? 
* How did you install UTM? 

Good day everyone i need your help  im trying to run kali linux on my M1 macbook machine using UTM  but it doesn't pass the black page, at first it was working just fine until i mistakenly deleted some file on my desktop since then it never worked again 

your help will be highly appreciated because i checked everywhere and i couldn't find the solution on this particular , a step by step solution will be highly appreciated  even if its a short video will help since i'll be able to follow through, i have tried everything i just can't get it right and it frustrates me.
<img width="945" alt="Screenshot 2022-08-19 at 22 52 11" src="https://user-images.githubusercontent.com/111610154/185706046-55e96b84-c0bc-4d39-83f1-72101a67fa9d.png">
<img width="789" alt="Screenshot 2022-08-19 at 22 52 46" src="https://user-images.githubusercontent.com/111610154/185706101-f7957df0-1b5a-4c52-b6cb-342b27b340c3.png">
<img width="788" alt="Screenshot 2022-08-19 at 22 53 09" src="https://user-images.githubusercontent.com/111610154/185706131-26cee880-96d2-45c2-b1a4-1d3aa047f0e9.png">
<img width="1097" alt="Screenshot 2022-08-19 at 22 53 56" src="https://user-images.githubusercontent.com/111610154/185706165-7925826a-e8f6-4743-a1c0-3cccb384f8af.png">

**Comment by ghost** [comment]
Try this: [#4090](https://github.com/utmapp/UTM/issues/4090#issuecomment-1213306172)

**Comment by VMGoodL** [comment]
Thank you so much it works perfect.

**Comment by optikfluffel** [comment]
I'm running version `4.0.8` of UTM and the mentioned "Full Graphics" and "Text only" options are gone. Is there any other way to get Kali to install?

**Reply by osy** [reply]
https://docs.getutm.app/guides/kali/#black-screen-on-start

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start
